app/uk/gov/hmrc/vulnerabilities/service/UpdateVulnerabilitiesService.scala

```diff
@@ -45,19 +46,41 @@ class UpdateVulnerabilitiesService @Inject()(
       //Only download reports that don't exist in last 7 days in our raw reports collection
       recentReports   <- rawReportsRepository.getReportsInLastXDays()
       outOfDateSVDeps  = whatsRunningWhereService.removeSVDIfRecentReportExists(svDeps, recentReports)
-      _               <- xrayService.processReports(outOfDateSVDeps)
-      _                = logger.info(s"Finished generating and inserting reports into the rawReports collection")
+      _ <- xrayService.processReports(outOfDateSVDeps)
+      _ = logger.info(s"Finished generating and inserting reports into the rawReports collection")
       //Transform raw reports to Vulnerability Summaries
-      unrefined       <- rawReportsRepository.getNewDistinctVulnerabilities()
-      _                = logger.info(s"Retrieved ${unrefined.length} unrefined vulnerability summaries")
-      reposWithTeams  <- teamsAndRepositoriesConnector.getCurrentReleases()
-      refined          = vulnerabilitiesService.convertToVulnerabilitySummary(unrefined, reposWithTeams, svDeps)
-      assessments     <- assessmentsRepository.getAssessments()
+      unrefined <- rawReportsRepository.getNewDistinctVulnerabilities()
+      _ = logger.info(s"Retrieved ${unrefined.length} unrefined vulnerability summaries")
+      reposWithTeams <- teamsAndRepositoriesConnector.getCurrentReleases()
+      refined = vulnerabilitiesService.convertToVulnerabilitySummary(unrefined, reposWithTeams, svDeps)
+      assessments <- assessmentsRepository.getAssessments()
       finalAssessments = assessments.map(a => a.id -> a).toMap
-      finalSummaries   = vulnerabilitiesService.addInvestigationsToSummaries(refined, finalAssessments)
-      _                = logger.info("About to delete all documents from the vulnerabilitySummaries repository")
+      finalSummaries = vulnerabilitiesService.addInvestigationsToSummaries(refined, finalAssessments)
+      _ = logger.info("About to delete all documents from the vulnerabilitySummaries repository")
       //Update final Collection
-      summariesCount  <- vulnerabilitySummariesRepository.deleteOldAndInsertNewSummaries(finalSummaries)
+      summariesCount <- vulnerabilitySummariesRepository.deleteOldAndInsertNewSummaries(finalSummaries)
+    } yield logger.info(s"Inserted ${summariesCount} documents into the vulnerabilitySummaries repository")
+  }
+
+
+  def updateVulnerabilities(serviceName: String,
```

Comment:
There's a lot of duplication with `updateVulnerabilities`. Maybe a common function `updateVulnerabilities(svDeps: Set[ServiceVersionDeployments])`? (The `DeploymentHandler` could probably just call that)

Reply:
Can we rename the first function to `updateAllVulnerabilities` to avoid overloading (else gets awkward to call functionally)

Reply:
Updated as requested